src/QAT/qat/purr/backends/verification.py

```diff
@@ -155,5 +157,8 @@ def verify_program(
     return execute(program, model, compiler_config)
 
 
+verification_engines: Dict[str, VerificationEngine] = {"Lucy": LucyVerificationEngine}
```

Comment:
If this sticks around it should use develops approach instead: https://github.com/oqc-community/qat/pull/48/files